test/e2e/utils.go

```diff
@@ -534,7 +497,13 @@ func defaultRouteNextHopInterface(node string) AsyncAssertion {
 		return gjson.ParseBytes(currentStateJSON(node)).Get(path).String()
 	}, 15*time.Second, 1*time.Second)
 }
+
 func vlan(node string, iface string) string {
 	vlanFilter := fmt.Sprintf("interfaces.#(name==\"%s\").vlan.id", iface)
 	return gjson.ParseBytes(currentStateJSON(node)).Get(vlanFilter).String()
 }
+
+func getTestName() string{
+	fileName := path.Base(CurrentGinkgoTestDescription().FileName)
```

Comment:
Can we get name of the specific test here?

Reply:
What do u mean? specific test case? not file?

Reply:
Yes, each `It` represented by a file. Now, when one of the tests fail, the log contains the whole module, so it may be harder to find the correct spot. But again, it all depends on the difficulty of such task.

Reply:
Not sure it so  easy. The easiest way is to run dump from AfterEach function of each test but then it will add run time for each case not sure it worth it.

Reply:
This is the kind of thing that get improved with use, let's merge it and ideas will pop up.